packages/sui-ssr/bin/sui-ssr-dev.js

```diff
@@ -63,6 +66,13 @@ const linkStatics = () => {
   )
 }
 
+const copyStatics = () => {
+  return Promise.allSettled([
+    cp(path.join(SRC_PATH, '404.html'), path.join(PUBLIC_OUTPUT_PATH, '404.html'), {recursive: true}),
+    cp(path.join(SRC_PATH, '500.html'), path.join(PUBLIC_OUTPUT_PATH, '500.html'), {recursive: true})
```

Comment:
this should be ok if one of the files (or both) doesn't exist, right?

Reply:
yep because allSettled is used